History.md

```diff
@@ -1,5 +1,10 @@
 ## v.NEXT
 
+* `observe`/`observeChanges` callbacks are now bound.
```

Comment:
Will try and elaborate a bit.

`this` isn't set to anything specific, and if we were to change it I don't know to what.